docs/about/starlite-releases.rst

```diff
@@ -38,12 +64,30 @@ won't be sweeping changes that require a complete rewrite of an application or m
 migration efforts when upgrading to a new major version.
 
 
-LTS releases and support
-------------------------
+Supported versions
+------------------
+
+Current version
+    The current version is the last release of the most recent major version. This
+    version is under active development and will receive bugfixes as well as feature
+    updates in minor releases (see `Version numbering`_)
+
+Maintenance versions
+    When a new major version is released, the last *minor* version before it enters
+    maintenance mode. It will receive bugfixes and other critical patches at least until
+    the next major version is released.
+
+LTS versions
+    The *last minor* release of every other *major version* will be an LTS release,
+    guaranteed to be supported for the duration of the next 2 major versions
```

Comment:
I'm not sure this is wise for us for now. I think maintenance as you called it is fine. 

Otherwise, we can say we will continue to support the last minor version for two major versions. 

So, `1.51.*` will be maintained until the release of `4.0.0`.

Reply:
Agree, Current and Maintenance sufficient for now.
 
We should guarantee a healthy minimum period for version maintenance, so that the number of versions we maintain will be a function of cadence of major releases we make.

So lets say we go from 2.0 -> 3.0 fairly quickly, but then it takes us 2 years to go from 3.0 -> 4.0, I don't think we should still have to maintain 1.51 in 2025 due to a rigid rule about # of major versions.

Reply:
> I don't think we should still have to maintain 1.51 in 2025 due to a rigid rule about # of major versions.

This would be addressed by the (semi) fixed release schedule for major releases.